### PR TITLE
Add escpae->escape and escpaed->escaped

### PR DESCRIPTION
**Pull request by EdwardBetts**

### CODE REVIEW

**Comment by larsoner** [comment]
Can you fix the merge conflict?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5978,6 +5978,8 @@ esacpe->escape
 esacpes->escapes
 escapeable->escapable
 escapemant->escapement
+escpae->escape
+escpaed->escaped
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: not highly likely but may also be `escapade` (link)](https://www.merriam-webster.com/dictionary/escapade)

**Comment by larsoner** [comment]
Needs merge/rebase

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: I think this is rare enough we should be okay either way

**Pull request merged**